tools/configure

```diff
@@ -2297,6 +2297,8 @@ case $build_cpu in #(
     OMR_BUILD_ARCH=x86 ;; #(
   x86_64) :
     OMR_BUILD_ARCH=x86 ;; #(
+  riscv64) :
+    OMR_BUILD_ARCH=riscv ;; #(
```

Comment:
This is a generated file which shouldn't be edited directly. You need to edit tools/configure.ac and rerun autoconf, just like the toplevel configure script.

Reply:
This file was already automatically generated based on `omr/tools/configure.ac` and `omrmakefiles/confighelpers.m4`
```
m4_include([../omrmakefiles/confighelpers.m4])
```
The changes at `omrmakefiles/confighelpers.m4` was placed at https://github.com/eclipse/omr/pull/4433
```
...
		[x86_64],[$1=x86],
		[x86_64],[$1=x86],
		[riscv64],[$1=riscv], <-----
```

Reply:
Ahh I see. I know it's a pain to move changes across commits, but I would appreciate it if the changes to `confighelpers.m4` were grouped into this PR.

Reply:
Already moved `confighelpers.m4` to this PR for your convenience  (the original change at #4433 has been removed).